src/isolate.rs

```diff
@@ -1303,6 +1398,29 @@ impl AsMut<Isolate> for Isolate {
   }
 }
 
+impl OwnedIsolate {
+  /// Creates a snapshot data blob.
+  /// This must not be called from within a handle scope.
+  ///
+  /// # Panics
+  ///
+  /// Panics if the isolate was not created using [`Isolate::snapshot_creator`]
+  #[inline(always)]
+  pub fn create_blob(
+    mut self,
+    function_code_handling: FunctionCodeHandling,
+  ) -> Option<StartupData> {
+    let mut snapshot_creator =
+      self.get_annex_mut().maybe_snapshot_creator.take().unwrap();
+    ScopeData::get_root_mut(&mut self);
```

Comment:
This call is unnecessary - the subsequent clear_scope_and_annex() call takes care
if it.

Reply:
Did that in https://github.com/denoland/rusty_v8/pull/973